src/gui/320x240x16/_gui.h

```diff
@@ -1,6 +1,15 @@
+#include "target.h"
+
+#ifndef LCD_WIDTH
 #define LCD_WIDTH 320
+#endif
+
+#ifndef LCD_HEIGHT
 #define LCD_HEIGHT 240
+#endif
```

Comment:
Lint Error:
```
5:  Uncommented text after #endif is non-standard.  Use a comment.  
9:  Uncommented text after #endif is non-standard.  Use a comment.  
```

Reply:
@PhracturedBlue what does this mean?

Reply:
check if there is a hidden character (or may the wrong line-feed?) after the #endif.  The code matched a non-whitespace, non-comment character after the #endif string on those lines